KarhooUISDK.podspec

```diff
@@ -28,5 +27,6 @@ Pod::Spec.new do |s|
   s.dependency    'Braintree/PaymentFlow', '~> 4.37'
   s.dependency    'PhoneNumberKit', '3.3.1'
   s.dependency    'Adyen', '3.6.0'
+  s.dependency    'SwiftLint'
```

Comment:
Why is this a dependency of this pod? It's just a tool to lint the code. The developer integrating the SDK should not import SwiftLint, if he/she does not want it.